packages/blockchain-wallet-v4-frontend/src/components/StepIndicator/index.js

```diff
@@ -50,8 +50,8 @@ const Steps = styled.div`
 
 const Step = styled.span`
   font-size: 14px;
-  min-width: ${props => props.coinify ? '135px' : '70px'};
-  max-width: ${props => props.coinify ? '135px' : '70px'};
+  min-width: ${props => props.minWidth ? '135px' : '70px'};
```

Comment:
Small nag but I would rather this be `props.minWidth || '70px'`. Or just `props.minWidth`, and make the property required. Same with maxWidth.